lib/Optimizer/Lower.cpp

```diff
@@ -219,6 +219,20 @@ static void lowerReluNode(Function *F, LoweredInfoMap *loweredMap,
   replaceAllUsesOfWith(loweredMap, R.getResult(), relu);
 }
 
+static void lowerPReluNode(Function *F, LoweredInfoMap *loweredMap,
+                           const PReluNode &R) {
+  // PRelu is :
+  // slope * x    if x < 0
+  // x            if x >= 0
+  // where slope is a variable.
```

Comment:
Good point. I meant that this is not necessarily a const but an input of a different function (which is what the description says now). The documentation doesn't state anything about if the values should be identical (https://github.com/onnx/onnx/blob/fb1a80692c1ab0bd27b1072f2e7bffacba336777/docs/Operators.md#PRelu). 
I changed the unit-test accordingly to check for a general case.

Reply:
I'm not sure "function" makes sense for us either since we have `Functions` that are essentially a container for series of Nodes 😄-- do you just mean the `slope` input is an output from another `Node`?

Reply:
Yes... Not used to the terminology. Will get there. Changing to "node".